packages/cli/src/help/help-extension.ts

```diff
@@ -55,8 +55,15 @@ export default class ChecklyHelpClass extends Help {
       this.log('')
     }
 
+    this.log(this.section('ENVIRONMENT VARIABLES',
+    `Running the CLI from your CI pipeline will need to export variables in the shell:\n
+  CHECKLY_ACCOUNT_ID    Checkly account ID.
+  CHECKLY_API_KEY       Checkly User API Key.
+  CHECKLY_ACCOUNT_NAME  (optional) Checkly account Name you want to use.
```

Comment:
Let's remove `CHECKLY_ACCOUNT_NAME`. That is only for local development so not needed for the user help

Reply:
Yes. I misunderstood why the `CHECKLY_ACCOUNT_NAME` is used. I'll remove it.